packages/xmlbuilder/package.js

```diff
@@ -1,16 +1,20 @@
 Package.describe({
   summary: "An XML builder for node.js similar to java-xmlbuilder.",
-  version: '2.5.15'
+  version: '2.5.16'
```

Comment:
This would need a major version bump, since the API is not compatible

Reply:
ok, if this package ends up being kept rather than a using the xmlbuilder2 node package directly ill make it 3.0.0